src/test/java/org/recap/service/executor/datadump/AbstractDataDumpExecutorServiceUT.java

```diff
@@ -108,7 +124,20 @@ public void testProcess() throws NoSuchMethodException, InvocationTargetExceptio
         }
 
     }
-    private String getDateTimeString() {
+
+    @Test
```

Comment:
Remove System.out.print
Why are we having getFormaatedDateString and getDateTimeString() methods in the service level? Can this be move to some util classes?